obspy/clients/nrl/tests/test_nrl.py

```diff
@@ -19,21 +19,16 @@ class NRLTestCase(unittest.TestCase):
     """
     def setUp(self):
         # Small subset of NRL included in tests/data
-        self.local_nrl_root = os.path.join(os.path.dirname(__file__),
-                                           'data',
-                                           'IRIS',
-                                           )
+        self.local_nrl_root = os.path.join(
+            os.path.dirname(__file__), 'data', 'IRIS')
         self.nrl_local = NRL(root=self.local_nrl_root)
         self.local_dl_key = ['REF TEK', 'RT 130 & 130-SMA', '1', '1']
         self.local_sensor_key = ['Guralp', 'CMG-3T', '120s - 50Hz', '1500']
 
+        # This is also the default URL.
         self.nrl_online = NRL(root='http://ds.iris.edu/NRL')
-        self.nrl_default = NRL()
```

Comment:
Online is default, however this tests calling the default with no args. Which Is valuable. I'd be ok leaving it out from the `list_of_nrls` that are functionally tested, but I think testing the creation should be kept.